rttr-de.po

```diff
@@ -1205,7 +1213,7 @@ msgstr "Wasser gefunden"
 
 #: gameData/BuildingConsts.cpp:183
 msgid "From the lookout tower you can see far into previously unexplored lands."
-msgstr "Für eine weite Aussicht in unerforschte Gebiete sorgt der Spähturm."
+msgstr "Der Spähturm bietet eine weite Aussicht in unerforschtes Gebiet. "
```

Comment:
klingt "in unerforschte Gebiete" nicht besser? oder waren das die originalen Übersetzungen?